src/features/profile/components/MyProfileGeneral/NSFWControls.tsx

```diff
@@ -21,7 +21,7 @@ export default function NSFWControls(): JSX.Element {
   return (
     <div className='w-full mt-20 mb-50px'>
       <div className='flex pt-3'>
-        <div className='text-gray-71 mr-4'>Hide Not Safe For Work NFTs</div>
+        <div className='text-gray-71 mr-4'>Show advanced features</div>
```

Comment:
Text is changed, but implementation remains the same, I think it's not correct. Could you change implementation as well, so "Hide Not Safe For Work" will be toggled by clicking on "Show advanced features"?

Reply:
I have changed implementation as well, not only text.
Please check again.
! [image](https://user-images.githubusercontent.com/64049500/131484635-9b5d479b-ea8a-495a-b6c3-3d038a8212a6.png)